allocator.py

```diff
@@ -0,0 +1,92 @@
+from people import Fellow, Staff
+from room import Office, LivingSpace
+import random
+
+class Dojo(object):
+    def __init__(self):
+        self.room_name = []
+        self.offices = {}
+        self.livings = {}
+
+        self.staff_list = []
+        self.fellow_list = []
+
+    def create_room(self, is_office, room_type):
+        rooms = room_type
```

Comment:
Does `room_type` here refer to the name to be given to the room or something else?